zoo/gobigger/env/gobigger_env.py

```diff
@@ -0,0 +1,500 @@
+import gym
+import numpy as np
+from ding.envs import BaseEnv, BaseEnvTimestep
+from ding.utils import ENV_REGISTRY
+from gobigger.envs import GoBiggerEnv
```

Comment:
add import check like [this](https://github.com/opendilab/DI-engine/blob/main/ding/utils/data/dataset.py#L128)

Reply:
done